aks-kubesphere-linux/azuredeploy.parameter.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "virtualNetworkName": {
+            "value": "KubeSphereRG-vnet"
```

Comment:
remove the literal values from here and use defaultValues in the template

Reply:
Oh THX, I will do it.